.tekton/serverless-bundle-135-pull-request.yaml

```diff
@@ -7,14 +7,13 @@ metadata:
     build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
-    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
-      == "main"  && ( files.all.exists(x, x.matches('^olm-catalog/serverless-operator/')) || files.all.exists(x, x.matches('^.tekton/')) )
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main" && ( files.all.exists(x, x.matches('^olm-catalog/serverless-operator/')) || files.all.exists(x, x.matches('^.tekton/')) )
   creationTimestamp: null
   labels:
     appstudio.openshift.io/application: serverless-operator-135
-    appstudio.openshift.io/component: serverless-bundle-main
+    appstudio.openshift.io/component: serverless-bundle-135
```

Comment:
@pierDipi this is strange, that this was working, even we renamed the component a couple of weeks ago (https://github.com/openshift-knative/hack/tree/main/.konflux/applications/serverless-operator-135/components and https://github.com/openshift-knative/hack/pull/334/commits/d15fad9f310aec6c0d34d630fd0fde6e40b2091d)

:thinking:

Reply:
maybe that's why we didn't see the EC run here?